source/pricing.blade.php

```diff
@@ -149,4 +93,52 @@ class="ud-single-pricing first-item{{ $content->isActive ? ' active' : ''}} wow
       </div>
     </div>
   </section>
+
+  <section id="pricing" class="ud-pricing">
+    <div class="container">
+
+      <h2 class="display-6 text-center mb-4">Compare plans</h2>
+
+      <div class="table-responsive">
+        <table class="table text-center">
+
+          <thead>
+            <tr>
+              <th style="width: 34%;"></th>
+              <th style="width: 22%;">{{ $page->t('Basic') }}</th>
+              <th style="width: 22%;">{{ $page->t('Business') }}</th>
+            </tr>
+          </thead>
+          @foreach($page->optionsServicesLibresign as $item => $optionList)
+          <tbody>
+            <tr>
+              <th scope="row" class="text-start">{{ $optionList->service }}</th>
+                @if($optionList->isBasic == true && $optionList->isBusiness == true)
+                  <td><i class="lni lni-checkmark"></i></td>
+                  <td><i class="lni lni-checkmark"></i></td>
+                @elseif($optionList->isBasic == true && $optionList->isBusiness == false)
+                  <td><i class="lni lni-checkmark"></i></td>
+                  <td><i class="lni lni-close"></i></td>
+                @elseif($optionList->isBasic == false && $optionList->isBusiness == true)
```

Comment:
I think that can't have booth

```suggestion
                @elseif($optionList->isBusiness == true)
```